crates/compiler/parse/src/pattern.rs

```diff
@@ -85,15 +85,16 @@ fn loc_pattern_help_help<'a>(
 ) -> impl Parser<'a, Loc<Pattern<'a>>, EPattern<'a>> {
     one_of!(
         specialize_err(EPattern::PInParens, loc_pattern_in_parens_help()),
+        loc(string_like_pattern_help()),
+        loc(number_pattern_help()),
         loc(underscore_pattern_help()),
-        loc_ident_pattern_help(can_have_arguments),
         loc(specialize_err(
             EPattern::Record,
             crate::pattern::record_pattern_help()
         )),
         loc(specialize_err(EPattern::List, list_pattern_help())),
-        loc(number_pattern_help()),
-        loc(string_like_pattern_help()),
+        // Move my pnc tag apply parser here
```

Comment:
This looks like a WIP comment, not meant to be committed? If not, maybe reword to make the intent clearer.